### PR TITLE
#185460884 fix user change password monthly 

### PR DESCRIPTION
**Pull request by jcharles921**
## What does this PR do?
This PR fix the login to check if a user do change his password after a month.
## Description of Task to be completed?
- run a cron job every month and send an email to the users whose password aren't changed every month
- check at login if the user change his password every month

## How should this be manually tested?

- Clone the repository using `git clone` [link](https://github.com/atlp-rwanda/e-comm-team-techsmith-bn)

- Checkout to the branch using `git checkout <branch-name>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm run dev`

## What are the relevant pivotal tracker/Trello stories?

[#185460884](https://www.pivotaltracker.com/story/show/185460884)

### CODE REVIEW

**Review by bergad37 — APPROVED** [review]
Nice work  to add getting all wishlist and check password expiration routes!

**Review by ReneNtabana — APPROVED** [review]

File: build/test/addProductToWishlist.test.js [hunk 1]
```diff
@@ -86,11 +86,16 @@ describe('delete single prouduct from wishlist', function () {
...
-    });
-  });
-});
\ No newline at end of file
+// describe('deleting all products in wishlist', () => {
+//   it('should delete all products in wishlist and http code 200', (done) => {
+//     chai
+//       .request(app)
+//       .delete('/api/wishlist/')
+//       .set('cookie', buyerCookie)
```
Comment (nishimweprince) [inline build/test/addProductToWishlist.test.js]: Please remove all these comments!

File: src/test/addProductToWishlist.test.js [hunk 2]
```diff
@@ -105,17 +105,17 @@ describe('delete single prouduct from wishlist', () => {
...
-  });
+// describe('deleting all products in wishlist', () => {
+//   it('should delete all products in wishlist and http code 200', (done) => {
+//     chai
+//       .request(app)
+//       .delete('/api/wishlist/')
+//       .set('cookie', buyerCookie)
+//       .end((err, res) => {
+//         expect(res).to.have.status(200);
+//         done();
```
Comment (nishimweprince) [inline src/test/addProductToWishlist.test.js]: Please remove these commented lines of code as well if they won't be needed.

**Review by nishimweprince — COMMENTED** [review]
Great work!! I however left comments suggesting to remove commented lines of code for your consideration.

**Review by Dyves1 — APPROVED** [review]

**Review by frankflej — APPROVED** [review]
Great work!

**Review by nishimweprince — APPROVED** [review]
Excellent work!!

**Review by niyontwali — APPROVED** [review]

**Review by niyontwali — APPROVED** [review]

**Review by nishimweprince — APPROVED** [review]
Excellent work!!

**Review by ReneNtabana — APPROVED** [review]

**Pull request merged**